src/main/java/org/folio/circulation/support/utils/DateTimeUtil.java

```diff
@@ -117,6 +133,23 @@ public static DateTime normalizeDateTime(DateTime dateTime) {
     return dateTime;
   }
 
+  /**
+   * Given a dateTime, normalize it.
+   * <p>
+   * For compatibility with JodaTime, when value is null, then a now() call
+   * via ClockUtil is used.
+   *
+   * @param dateTime The dateTime to normalize.
+   * @return The provided dateTime or if dateTime is null then ClockUtil.getZonedDateTime().
+   */
+  public static Instant normalizeDateTime(Instant dateTime) {
+    if (dateTime == null) {
+      return ClockUtil.getZonedDateTime().toInstant();
+    }
+
+    return dateTime;
+  }
```

Comment:
What is meant by normalization?

The implementation seems to return the current date and time if the value is null. To me, that is defaulting.

Reply:
This is described in the JavaDocs documentation.
In otherwords, by normalization, this will normalize the JavaTime behavior to be consistent with the JodaTime behavior.

JodaTime treats `null` as a call to `now()`, but JavaTime explicitly does not do this and treats `null` differently.
This is a breaking behavior of the JodaTime to the JavaTime that the normalization functions are provided to ease the transition.

I think there is a good reasons to bring up after the JodaTime to JavaTime conversion is completed to guarantee certain `null` behavior (as far as time is concerned).
It would be great to not have/need these normalization functions at all.
But for now, without them, the transition from JodaTime into JavaTime will result in numerous failures.

Reply:
> In otherwords, by normalization, this will normalize the JavaTime behavior to be consistent with the JodaTime behavior.

And that behaviour is to default to the clocks definition of now. Have I understood that correctly?

Reply:
> But for now, without them, the transition from JodaTime into JavaTime will result in numerous failures.

I'm not arguing for not doing this, I'm trying to understand the choice of naming.

Reply:
```
And that behaviour is to default to the clocks definition of now. Have I understood that correctly?
```
That is correct.

Reply:
My suggestion would be to name it after what it does (default to now). There could still be a comment explaining why this is important (to mimic the behaviour of JodaTime).

```suggestion
  public static Instant defaultToNow(Instant dateTime) {
    if (dateTime == null) {
      return ClockUtil.getZonedDateTime().toInstant();
    }

    return dateTime;
  }
```